index.html

```diff
@@ -1768,6 +1777,105 @@ <h2>Security Considerations</h2>
+resulting in reduced security. Ideally, cryptographic suites are implemented
+in software as specific, acceptable profiles of cryptographic parameters instead.
+        </p>
+        <p>
+Numbered versioning might specify a major and minor version number such as
+`1.0` or `2.1`. Numbered versioning conveys a specific order and suggests that
+higher version numbers are more capable than lower version numbers. The benefit
+of this approach is that it removes complex parameters that less expert
+developers might not understand with a simpler model that conveys that an
+upgrade might be appropriate. The drawback of this approach is that its not
+clear if an upgrade is necessary, as software version number increases often
+don't require an upgrade for the software to continue functioning. This can
+lead to developers thinking their usage of a particular version is safe, when
+it is not. Ideally, additional signals would be given to developers that use
+cryptographic suites in their software that periodic reviews of those
+suites for continued security are required.
+        </p>
+        <p>
+Date-based versioning specifies a particular release date for a specific
+cryptographic suite. The benefit of a date, such as a year, is that it is
+immediately clear to a developer if the date is relatively old or new. Seeing
+an old date might prompt the developer to go searching for a newer
+cryptographic suite, where as a parametric or number-based versioning scheme
+might not. The downside of a date-based version is that some cryptographic
+suites might not expire for 5-10 years, prompting the developer to go
+searching for a newer cryptographic suite only to not find one that is newer.
+While this might be an inconvenience, it is one that results in safer
+ecosystem behavior.
+        </p>
+        <p class="issue" data-number="38">
+The following text is currently under debate:<br><br>
+It is highly encouraged that cryptographic suite identifiers are versioned
+using a year designation. For example, the cryptographic suite identifier
+`ecdsa-2022` implies that the suite is probably an acceptable of ECDSA in the
+year 2025, but might not be a safe choice in the year 2042. A date-based
+versioning mechanism, however, is not enough by itself. All cryptographic
+suites that follow this specification are intended to be registered
+[[?VC-EXTENSION-REGISTRY]] in a way that clearly signal which cryptosuites
+are deprecated, standardized, or experimental. Cryptosuite registration will
+follow CFRG, IETF, NIST, FIPS, and safecurves guidance. Use of deprecated suites
+are expected to throw errors in implementations unless a `useUnsafeCryptosuites`
+option is used specifying exactly the unsafe cryptosuite to use.
+Use of experimental suites are expected to throw errors in implementations
+unless a `useExperimentalCryptosuites` option is used used specifying exactly
+the experimental cryptosuite to use.
```

Comment:
this is excellent!

Reply:
+1, but typo with repeated word: "used used".

Reply:
Typo fixed in latest PR.